.env.example

```diff
@@ -24,8 +24,8 @@ FINALIZATION_PERIOD_SECONDS=12
 ENABLE_GOVERNANCE=false
 
 # Finality gadget RPC
-# note: use external host address to avoid cross-container calls
-BBN_FINALITY_GADGET_RPC=11.22.33.44:50051
+# note: keep empty string before the Finality Gadget is up
```

Comment:
since we moved it, let's add a comment of example value 11.22.33.44:50051 and mentioned this is gRPC?